web3/utils/abi.py

```diff
@@ -78,6 +78,12 @@ def is_encodable(_type, value):
 
         max_length = int(sub)
         return len(value) <= max_length
+    elif base == 'address':
+        if not is_string(value):
```

Comment:
Do you mind changing this to use the `is_address` utility function found at `web3.utils.address.is_address`?